pages/docs/learn/inngest-functions.mdx

```diff
@@ -0,0 +1,104 @@
+import IconConcurrency from 'src/shared/Icons/FlowControl/Concurrency';
+
+export const description = 'Learn what Inngest functions are and of what they are capable.';
+
+# Inngest Functions
+
+Inngest functions enable developers to create and manage event-driven workflows. They offer a flexible way to build and manage event-driven workflows and background tasks, providing robust tools for retrying, scheduling, and coordinating complex sequences of operations.
```

Comment:
**suggestion**: While accurate, I don't know if we want to push the "event-driven workflows" term hard here (and also not twice). I might instead share what it allows a developer achieve like "run reliable background logic, from background jobs to complex workflows." We don't push "event-driven" on our site or other places so I'd rather describe the benefits/outcomes and then explain how it works.

> Inngest functions enable developers run reliable background logic, from background jobs to complex workflows